Vector.cpp

```diff
@@ -1,8 +1,21 @@
 #include "Vector.h"
 #include <assert.h>
+#include <iostream>
+
+using namespace std;
 
 namespace Micromouse
 {
+	direction operator%(const direction &dir1, const direction &dir2)
```

Comment:
It has been suggested to change it to an overload of dir + dir so it is more obvious what it does. Also then dir - dir could be added. What do you think @RosyMapleMoth